src/tests/mocks/handlers/clinicalTrialsGov.handler.ts

```diff
@@ -0,0 +1,360 @@
+import { rest } from 'msw';
+import * as process from 'node:process';
+
+const BASE_URL = 'https://clinicaltrials.gov';
+
+export const clinicalTrialsGovHandlers = [
+  rest.get(`${BASE_URL}/api/v2/studies?pageSize=1`, (request, response, context) => {
+    return response(context.body(JSON.stringify(responseJSON)));
+  }),
+
+  // rest.get(getStorageSignedUrl(':filename'), (request, response, context) => {
+  //   const file = new Blob(['Hello, world!'], { type: 'text/plain' });
+  //   return response(
+  //     context.set('Content-Length', file.size.toString()),
+  //     context.set('Content-Type', 'text/plain'),
+  //     context.set('Content-Disposition', 'inline'),
+  //     context.body(file)
+  //   );
+  // }),
+];
+
+const responseJSON = {
```

Comment:
1. extract json into separate file
2. remove properties that your service does not use -> you will start using this file when investigating new things instead of the real thing so you will shoot yourself in the foot if the api you are mocking would change in the meantime